source/catalog.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Каталог Седона</title>
+  </head>
+  <body class="page_ _body">
+    <header class="main-header">
+      <nav class="main-nav">
+      <div class="main-nav_ _wrapper">
+        <ul class="main-nav_ _list site-list">
+        <li class="site-list_ _item site-list_ _item-current "><a href="#">ГЛАВНАЯ</a>
```

Comment:
Тут для текущей страницы стоит использовать модификатор и опять же пробелы
```suggestion
        <li class="site-list__item site-list__item--current "><a href="#">ГЛАВНАЯ</a>
```